doc/src/fpga_api/prog_guide/readme.md

```diff
@@ -172,7 +172,7 @@ The table below groups important API calls by their functionality. For more info
 |           | ```fpgaReadMMIO[32, 64]()``` | Yes| Yes|Read a 32-bit or 64-bit value from MMIO space |
 |           | ```fpgaWriteMMIO[32, 64]()``` |Yes| Yes| Write a 32-bit or 64-bit value to MMIO space |
 |Memory management: Shared memory | ```fpga[Prepare, Release]Buffer()``` |Yes| Yes| Manage memory buffer shared between the calling process and an accelerator |
-|              | ```fpgaGetIOVA()``` | Yes| Yes|Return the virtual address of a shared memory buffer |
+|              | ```fpgaGetIOAddress()``` | Yes| Yes|Return the virtual address of a shared memory buffer |
```

Comment:
Struggling with "virtual address" here.  It's the IO/memory address that is being fetched.